dagshub/auth/oauth.py

```diff
@@ -52,9 +56,53 @@ def oauth_flow(
             f"Error while getting OAuth token: HTTP {res.status_code}, Body: {res.json()}"
         )
     token = res.json()
+
     logger.debug(f"Got token: {token}")
+    # Guy- should we print so massage to let the user know he is now logged in?
```

Comment:
That's what the logger.debug line does - don't add anything on top of it, that's just polluting the user's output